GeocachingGroupingUICache/src/main/com/gg/uicache/config/JMSConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.gg.uicache.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jms.support.converter.MappingJackson2MessageConverter;
+import org.springframework.jms.support.converter.MessageConverter;
+import org.springframework.jms.support.converter.MessageType;
+
+@Configuration
+public class JMSConfig {
+
+    @Bean // Serialize message content to json using TextMessage
+    public MessageConverter jacksonJmsMessageConverter() {
+        MappingJackson2MessageConverter converter = new MappingJackson2MessageConverter();
+        converter.setTargetType(MessageType.TEXT);
+        converter.setTypeIdPropertyName("_type");
```

Comment:
What is "_type"? Could be a constant with some reasoning why to be used.

Reply:
fixed

Reply:
fixed